### PR TITLE
Function Wrapper to parse Datetimes

### PR DESCRIPTION
**Pull request by shaardie**
This patch introduces a function wrapper to parse function argument strings to
datetime.datetime object and replace them.

With this function wrapper in place in the API it is hopefully easier to reuse
the parsing in different endpoints.

Signed-off-by: Sven Haardiek <sven@haardiek.de>

### CODE REVIEW

File: emissionsapi/web.py [hunk 1]
```diff
@@ -19,6 +19,31 @@
 logger = logging.getLogger(__name__)
 
 
+def parse_date(key):
```
Comment (lkiesow) [inline emissionsapi/web.py]: What about using `def parse_date(*key):` instead? That way you could use something like `@parse_date('begin', 'end`)`?

Comment (shaardie) [reply emissionsapi/web.py]: Great idea. I have done that.

File: emissionsapi/web.py [hunk 2]
```diff
@@ -19,6 +19,32 @@
 logger = logging.getLogger(__name__)
 
 
+def parse_date(*keys):
+    """Wrapper for function to replace dates inputed as strings as
```
Comment (lkiesow) [inline emissionsapi/web.py]:
```suggestion
    """Function wrapper replacing string date arguments with
```

File: emissionsapi/web.py [hunk 3]
```diff
@@ -19,6 +19,32 @@
...
+    datetime.datetime
+
+    :param keys: keys which are parsed and replaced
```
Comment (lkiesow) [inline emissionsapi/web.py]:
```suggestion
    :param keys: keys to be parsed and replaced
```

File: emissionsapi/web.py [hunk 4]
```diff
@@ -19,6 +19,32 @@
...
+    :type keys: array
```
Comment (lkiesow) [inline emissionsapi/web.py]:
```suggestion
    :type keys: list
```

**Pull request merged**